HISTORY.md

```diff
@@ -1,3 +1,90 @@
+# TileDB v2.19.0 Release Notes
```

Comment:
Should it have a paragraph 'Ongoing Development' or 'TileDB v2.20.0 (unreleased)' or something corresponding to the version on the branch?

Reply:
I will consider it moving forward, thanks!